LORIS_indepth.html

```diff
@@ -138,141 +67,43 @@ <h4 align="center"><font color="f27c30">Heterogenous data acquisition</font></h4
 			<h4 align="center"><font color="f27c30">Storage, processing & dissemination</font></h4>
 			</td>
 		</tr></table>
-
-	<img src="images/loris_versatility.png" align="center" width="90%" style="border:0">
-
-<!--	<table class = "reveal" style="padding:0;margin:0" width = "60%">
-		<tr>
-			<td width="25%" style="vertical-align:middle">
-				<img src="images/multisite_study.png" style="border:0">
-			</td>
-			<td width="25%" style="vertical-align:middle">
-				<img src="images/longitudinal.png" style="border:0">
-			</td>
-			<td width="25%" style="vertical-align:middle">
-				<img src="images/clinical_research.png" style="border:0">
-			</td>
-			<td width="25%" style="vertical-align:middle">
-				<img src="images/public_dataset.png" style="border:0">
-			</td>
-			</tr></table> -->
-          </section>
-
-<!-- WHAT DOES LORIS DO? -->
-<section>
-            <h2>What does LORIS do?</h2>
-            <img src="images/LORIS_functionality.png">
-          </section>
-
-<!-- PERFERRED SLIDE WHAT DOES LORIS DO -->	  
-	  <section>
+            <img src="images/loris_versatility.png" align="center" width="90%" style="border:0">
+            <p><small>A flexible, customizable, and <font color="f27c30">open source</font> neuroimaging <br> database suitable for multiple types of projects.</small></p>
+            </section>          
+              </section>
+              
+              <!-- What does LORIS do? Column 2 -->
+              
+        <section>
+               <section>
 	    <h2>What does LORIS do?</h2>
-	    <img src="images/loris_purpose.jpg" style="border:0" width="80%"> 
+	    <img src="images/LORIS_hexagons.png"> 
```

Comment:
@knoll-alex   could you enlarge the image so that it's at least 150% of current size?  the words inside the hexagons need to be bigger to be readable

Reply:
I tried - my preview doesn't seem to be working properly... can you confirm if the resizing worked?

Reply:
150% was too big - it doesn't fit on the page. I tried to make it as big as possible while still fitting on the page. let me know if it's ok?

Reply:
- [ ] @christinerogers ask Derek to make a bigger and transparent version (w/o orange squiggle